dask_snowflake/core.py

```diff
@@ -120,8 +120,8 @@ def to_snowflake(
     )
 
 
-def _fetch_batch(chunk: ArrowResultBatch, arrow_options: dict):
-    return chunk.to_pandas(**arrow_options)
+def _fetch_batches(chunks: list[ArrowResultBatch], arrow_options: dict):
+    return pd.concat([chunk.to_pandas(**arrow_options) for chunk in chunks], axis=0)
```

Comment:
I can imagine some Arrow concatenation first would make sense, but I'm happy to wait for this to come up in profiles before thinking about it.